src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -83,7 +83,9 @@ public sealed class Arguments : ArgumentsBase
         /// <param name="memorySize">Memory size for <see cref="LogisticRegression"/>. Lower=faster, less accurate.</param>
         /// <param name="optimizationTolerance">Threshold for optimizer convergence.</param>
         /// <param name="advancedSettings">A delegate to apply all the advanced arguments to the algorithm.</param>
-        public MulticlassLogisticRegression(IHostEnvironment env, string featureColumn, string labelColumn,
+        public MulticlassLogisticRegression(IHostEnvironment env,
+            string labelColumn,
```

Comment:
>labelColumn [](start = 19, length = 11)

it used to be `label` and `features` in other trainers, right?

Reply:
Thanks for this. 

---
In reply to: [231231728](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231231728) [](ancestors = 231231728)

Reply:
but should we have it consistent?

---
In reply to: [231244854](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231244854) [](ancestors = 231244854,231231728)

Reply:
for sure. I changed them to be "label, features, weights". I personally lean more towards "labelColum, featureColumn and weightColumn" 

---
In reply to: [231367774](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231367774) [](ancestors = 231367774,231244854,231231728)

Reply:
I also find labelColumn, featureColumn, weightColumn more appealing. I find them more representative of what the arguments specify (the name of these columns).

Reply:
FWIW, labelColumn, featureColumn, weightColumn sound better to me.